webapp/src/ts/modules/contacts/contacts.component.ts

```diff
@@ -263,17 +267,21 @@ export class ContactsComponent implements OnInit, OnDestroy{
   }
 
   private getChildren() {
-    let p;
     if (this.usersHomePlace) {
       // backwards compatibility with pre-flexible hierarchy users
       const homeType = this.usersHomePlace.contact_type || this.usersHomePlace.type;
-      p = this.contactTypesService.getChildren(homeType);
-    } else if (this.isAdmin) {
-      p = this.contactTypesService.getChildren();
-    } else {
-      return Promise.resolve([]);
+      return this.contactTypesService
+        .getChildren(homeType)
+        .then(children => children.filter(child => !child.person));
```

Comment:
Maybe we can make this piece of code reusable, like the old code did). 
```
const filterPlaceTypes = children => child => !child.person;
```

Reply:
Sure, I'll provide a function that will filter. It's going to be tiny cute one